crates/vm2/src/callframe.rs

```diff
@@ -133,17 +135,29 @@ impl<T, W> Callframe<T, W> {
         })
     }
 
+    /// Gets a raw inferred program counter. This value can be garbage if the frame is on an invalid instruction or free panic.
+    #[allow(clippy::cast_possible_wrap)] // false positive: `Instruction` isn't that large
+    pub(crate) fn get_raw_pc(&self) -> isize {
```

Comment:
I've tested performance relative to `offset_from` locally and there doesn't seem to be significant difference between the two.

Reply:
Well spotted! Looks like I forgot to read why offset_from is unsafe.